RecoTauTag/RecoTau/plugins/PFRecoTauProducer.cc

```diff
@@ -189,7 +189,7 @@ PFRecoTauProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions
   desc.add<double>("smearedPVsigmaZ", 0.005);
   desc.add<double>("MatchingConeSize_max", 0.6);
   desc.add<double>("HCALIsolConeSize_min", 0.0);
-  descriptions.add("pfRecoTauProducer", desc);
+  descriptions.add("PFRecoTauProducerDef", desc);
```

Comment:
the default module instance naming convention is lower case.
Let's keep it here as well.
As soon as one upper case use pops up, it gets copied all around

Reply:
ok, now renamed with lowercase